lms/static/js/student_account/views/RegisterView.js

```diff
@@ -4,29 +4,49 @@
         'jquery',
         'underscore',
         'gettext',
+        'edx-ui-toolkit/js/utils/string-utils',
         'js/student_account/views/FormView',
         'text!templates/student_account/form_status.underscore'
     ],
-        function($, _, gettext, FormView, formStatusTpl) {
+        function(
+            $, _, gettext,
+            StringUtils,
+            FormView,
+            formStatusTpl
+        ) {
             return FormView.extend({
                 el: '#register-form',
-
                 tpl: '#register-tpl',
-
+                validationUrl: '/api/user/v1/validation/registration',
                 events: {
                     'click .js-register': 'submitForm',
-                    'click .login-provider': 'thirdPartyAuth'
+                    'click .login-provider': 'thirdPartyAuth',
+                    'click input[required][type="checkbox"]': 'liveValidateHandler',
+                    'blur input[required]': 'liveValidateHandler',
+                    'focus input[required]': 'handleRequiredInputFocus',
+                // Select boxes are unfortunately special cases,
+                // because one "focuses" on the first item, not the input box.
+                    'blur select[required]:first': 'liveValidateHandler',
+                    'focus select[required]:first': 'handleRequiredInputFocus'
```

Comment:
Validation for select fields will only happen after you blur from your selection of the list. Selecting an item does not unfortunately count as blurring from the list -- you have to explicitly select the item, then click away from the list.

As for which child to choose, as far as I've seen, looking for blur/focus events on the first child has been sufficient for no matter what child you select.